client/dashboard/sites/site-fields.tsx

```diff
@@ -1,9 +1,41 @@
 import { useQuery } from '@tanstack/react-query';
 import { useInView } from 'react-intersection-observer';
+import { siteEngagementStatsQuery } from '../app/queries/site-stats';
 import { siteUptimeQuery } from '../app/queries/site-uptime';
 import { TextBlur } from '../components/text-blur';
 import type { Site } from '../data/types';
 
+export function EngagementStat( {
+	site,
+	type,
+}: {
+	site: Site;
+	type: 'visitors' | 'views' | 'likes';
+} ) {
+	const { ref, inView } = useInView( { triggerOnce: true, fallbackInView: true } );
+	const isEligible =
+		! site.is_deleted && ( ! site.jetpack || site.jetpack_modules?.includes( 'stats' ) );
+
+	const { data: stats, isLoading } = useQuery( {
+		...siteEngagementStatsQuery( site.ID ),
+		enabled: isEligible && inView,
+	} );
+
+	if ( ! isEligible ) {
+		return '-';
```

Comment:
We should display a dash - in `gray 400` as the empty state if the column isn't supported. See p1750235555750989/1750233879.494839-slack-C08JZTRS6NA